utilities_common/bgp_util.py

```diff
@@ -12,27 +12,27 @@
 from utilities_common import constants
 
 
-def is_ipv4_address(ipaddress):
+def is_ipv4_address(ipaddr):
```

Comment:
I want to say `ipaddr` is also a Python module (and I think we might also use it in some parts of the code base?). Maybe to be safe we should go with `ip_address` or just `address`.

Reply:
Thanks, updated to `ip_address`.